production/helm/loki/templates/networkpolicy.yaml

```diff
@@ -108,7 +108,6 @@ spec:
       {{- end }}
     {{- end }}
 
-{{- if .Values.ruler.enabled }}
```

Comment:
Thanks for addressing this. I'm not sure this is the fix though. I think there should be a default for the ruler instead.